File: workshops/lopeza/Unit 2/WS09NodeREST/models/vehicle.js
const mongoose = require('mongoose')

const vehicleSchema = new mongoose.Schema({
  plateId: {
    type: String,
    required: true,
    unique: true
  },
  brand: {
    type: String,
    required: true
  },
  model: {
    type: String,
    required: true
  },
  year: {
    type: Number,
    required: true
  },
  color: {
    type: String,
    required: true
  }

}, { collection: 'Vehicles' })

module.exports = mongoose.model('Vehicles', vehicleSchema)

File: assignments/lopeza/Unit 2/HW10ExternalUriLanguage/schemas/vehicle.py
def vehicleEntity(item) -> dict:
    return {
        "id": str(item["_id"]),
        "plateId": item["plateId"],
        "brand": item["brand"],
        "model": item["model"],
        "year": item["year"]
    }

def vehiclesEntity(entity) -> list: 
    return [vehicleEntity(item) for item in entity]